src/maintenance/setup.py

```diff
@@ -1,40 +1,57 @@
+#!/usr/bin/env python
+
 # --------------------------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
 
+from codecs import open
 from setuptools import setup, find_packages
 
+# HISTORY.rst entry.
+VERSION = '0.1.0'
```

Comment:
Is it expected that the version is downgraded from 1.0.1 to 0.1.0?

Reply:
yes, I wasn't sure about this, thanks for confirming.

Reply:
You can use new version "1.1.0"

---
In reply to: [478221129](https://github.com/Azure/azure-cli-extensions/pull/2214#discussion_r478221129) [](ancestors = 478221129)